errors/errctx.go

```diff
@@ -21,6 +21,10 @@ func (err ErrCtx) Ctx() context.Context {
 	return err.ctx
 }
 
+func Newf(ctx context.Context, format string, args ...interface{}) error {
```

Comment:
Ack 
In addition, the [Aurélien's comment](https://github.com/Scalingo/logs-archive-service/pull/135#pullrequestreview-1215515813) make me think: should we add others methods `New`, `Note`, `Wrap` (instead of `f` versions)?

Reply:
As my comment may suggest, I would encourage to do so :sweat_smile: